Dockerfile

```
@@ -1,72 +1,20 @@
-# https://nextjs.org/docs/advanced-features/output-file-tracing
-COPY --from=builder --chown=nextjs:nodejs /app/.next/standalone ./
-COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
-
-USER nextjs
-
-EXPOSE 3000
+COPY . /app
+WORKDIR /app
 
-ENV PORT 3000
+RUN cp .env env.sh && sed -i 's/^/export /g' env.sh && source env.sh
```

Hadolint (2.12.0)

[info] Not following: File not included in mock.

(SC1091)